android/quest/src/test/java/org/smartregister/fhirecore/quest/ui/patient/register/PatientRegisterFragmentTest.kt

```diff
@@ -74,6 +74,14 @@ class PatientRegisterFragmentTest : RobolectricTest() {
         "Razi"
       )
     )
+
+    Assert.assertTrue(
+      registerFragment.performFilter(
+        RegisterFilterType.SEARCH_FILTER,
+        PatientItem(id = "1234"),
+        "1234"
+      )
+    )
```

Comment:
Flip the actual and expected result for this assertion.

Reply:
@ellykits This is an `assertTrue` statement which takes in one argument.

Reply:
Haha damn me! :joy: Noted approved.